dashboard/pkg/epinio/pages/auth/verify.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+import Vue from 'vue';
+import epinioAuth from '../../utils/auth';
+import Banner from '@components/Banner/Banner.vue';
+
+interface Data {
+  error: string,
+}
+
+export default Vue.extend<Data, any, any, any>({
+
+  components: { Banner },
+
+  data() {
+    return { error: '' };
+  },
+
+  async fetch({ store, route }: { store: any, route: any}) {
+    const {
+      error, error_description: errorDescription, errorCode, errorMsg
```

Comment:
`error_description` can be removed

Reply:
I've removed errorCode and errorMsg, as these weren't used by dex or the dep to deal with oidc. `error_description` is a valid param though